conftest.py

```diff
@@ -0,0 +1,66 @@
+import pytest
```

Comment:
Aren't we doing the same [here](https://github.com/opensearch-project/opensearch-py-ml/blob/main/tests/conftest.py#L162-L164)? May be we can add setup this iris data set over there?